### PR TITLE
Fixes https://github.com/Open-Systems-Pharmacology/Matlab-Toolbox/issues/43 Update call to MatalbIndividualFactory and MatlabPopulationFactory to use new namespaces

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

**Comment by Yuri05** [comment]
Interface of PKSimCreateIndividual, PKSimCreatePopulation and DefaultPopulationSettings not changed
(DefaultPopulationSettings  just extended with units), thus existing matlab code calling the functions should work without changes

File: code/auxiliaries/DefaultPopulationSettings.m [hunk 1]
```diff
@@ -17,30 +17,37 @@
...
-    populationSettings = PKSim.Core.Batch.PopulationSettings;
     gestationalAgeRange = PKSim.Core.CoreConstants.PretermRange;
 
     populationSettings.Species             ='Human';
     populationSettings.Population          ='European_ICRP_2002';
+    
     populationSettings.MinAge              = NaN;
     populationSettings.MaxAge              = NaN;
+    populationSettings.AgeUnit             = 'year(s)';
+
```
Comment (Yuri05) [inline code/auxiliaries/DefaultPopulationSettings.m]: Extended with units for age, weight, height etc.
Per default all units are set to internal units for the corresponding dimensions

File: code/auxiliaries/PKSimCreateIndividual.m [hunk 2]
```diff
@@ -155,15 +155,19 @@
...
+    originData.Gender         = gender;
+    originData.Age            = parameterFrom(age, 'year(s)');
+    originData.GestationalAge = parameterFrom(gestational_age, 'week(s)');
+    originData.Weight         = parameterFrom(weight, 'kg');
+    originData.Height         = parameterFrom(height, 'dm');
+    
+    calculationMethods        = NET.createArray('System.String', 1);
+    calculationMethods(1)     = endothelialSurfaceAreaCalculationMethod;
+    AddCalculationMethods(originData, calculationMethods);
+    %originData.AddCalculationMethod('SurfaceAreaPlsInt',endothelialSurfaceAreaCalculationMethod);
```
Comment (Yuri05) [inline code/auxiliaries/PKSimCreateIndividual.m]: adjusted to new PKSim namespaces and calls

File: code/auxiliaries/PKSimCreatePopulation.m [hunk 3]
```diff
@@ -25,7 +25,8 @@
     ontogenies = NET.createArray('System.String', 0);
     
     %---- create new population
-    result=PopulationFactory.CreatePopulation(populationSettings, ontogenies);
+    PKSimPopulationSettings = pksimPopulationSettingsFrom(populationSettings);
+    result=PopulationFactory.CreatePopulation(PKSimPopulationSettings, ontogenies);
```
Comment (Yuri05) [inline code/auxiliaries/PKSimCreatePopulation.m]: get new PKSim population settings from Matlab structure

**Comment by msevestre** [comment]
Interface of PKSimCreateIndividual, PKSimCreatePopulationand DefaultPopulationSettings not changed??

I think you are looking at the wrong implementation here. Everything has changed. 

```
 public class PopulationSettings
   {
      [Required]
      public int NumberOfIndividuals { get; set; }
      public int? ProportionOfFemales { get; set; }
      public ParameterRange Age { get; set; }
      public ParameterRange Weight { get; set; }
      public ParameterRange Height { get; set; }
      public ParameterRange GestationalAge { get; set; }
      public ParameterRange BMI { get; set; }

      [Required]
      public Individual Individual { get; set; }
   }
```

There is no MinWeight, MinHeight etc.. Or what are objects are you referering to?

File: code/auxiliaries/PKSimCreatePopulation.m [hunk 4]
```diff
@@ -59,9 +60,43 @@
...
     end
 end
+
+function PKSimPopulationSettings = pksimPopulationSettingsFrom(populationSettings)
+    PKSimPopulationSettings = PKSim.Core.Snapshots.PopulationSettings;
+    
+    PKSimPopulationSettings.NumberOfIndividuals = populationSettings.NumberOfIndividuals;
+    PKSimPopulationSettings.ProportionOfFemales = populationSettings.ProportionOfFemales;
+    
+    PKSimPopulationSettings.Age = parameterRangeFrom(populationSettings.MinAge, populationSettings.MaxAge, populationSettings.AgeUnit);
```
Comment (msevestre) [inline code/auxiliaries/PKSimCreatePopulation.m]: oh I see.  populationSettings is now a Matlab objecT?

**Comment by Yuri05** [comment]
I'm talking about the **_Matlab_** Interface here

File: code/auxiliaries/DefaultPopulationSettings.m [hunk 5]
```diff
@@ -6,7 +6,7 @@
 %       is used in function PKSimCreatePopulation   
 
 % Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org 
-% Date: 12-jan-2017
+% Date: 30-apr-2018
```
Comment (msevestre) [inline code/auxiliaries/DefaultPopulationSettings.m]: Why do we have dates here? This can only go wrong

Comment (msevestre) [reply code/auxiliaries/DefaultPopulationSettings.m]: My suggestion would be to remove dates in those files. This does not make much sense to add a date in a source code file

File: code/processSimulation.m [hunk 6]
```diff
@@ -1,126 +1,126 @@
...
+% success=processSimulation('*');
+%
+% see also INITSIMULATION
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+% Date: 24-Sep-2010
+
+global DCI_INFO;
+global MOBI_SETTINGS;
+
+%% check Inputs --------------------------------------------------------------
```
Comment (msevestre) [inline code/processSimulation.m]: some weird ^M at the end here. Formatting issues probably

**Pull request merged**